core/arch/arm/plat-imx/config/imx6ul.h

```diff
@@ -0,0 +1,64 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2017-2018 NXP
+ */
+
+#ifndef PLATFORM_CONFIG_IMX6UL_H
+#define PLATFORM_CONFIG_IMX6UL_H
+
+#ifdef CFG_WITH_PAGER
+#error "Pager not supported for platform mx6ulevk"
+#endif
+#ifdef CFG_WITH_LPAE
+#error "i.MX 6UL/6ULL does not support LPAE"
+#endif
+
+#define CFG_TEE_CORE_NB_CORE		1
+
+#define DRAM0_SIZE		CFG_DDR_SIZE
+#define DDR_SIZE			DRAM0_SIZE
+#define DDR_PHYS_START			DRAM0_BASE
+#define CFG_DDR_START			DDR_PHYS_START
+
+#if defined(PLATFORM_FLAVOR_mx6ul9x9evk)
+#define CFG_DDR_TEETZ_RESERVED_START	0x8E000000
+#endif
+
+#ifndef CFG_DDR_TEETZ_RESERVED_START
+#define CFG_DDR_TEETZ_RESERVED_START	0x9E000000
+#endif
+
+#define CFG_DDR_TEETZ_RESERVED_SIZE	0x02000000
+
+#define CFG_PUB_RAM_SIZE	(2 * 1024 * 1024)
+
+#define TZDRAM_BASE		(CFG_DDR_TEETZ_RESERVED_START)
+#define TZDRAM_SIZE		(CFG_DDR_TEETZ_RESERVED_SIZE - \
+				 CFG_PUB_RAM_SIZE)
+
+#define CFG_SHMEM_START		(CFG_DDR_TEETZ_RESERVED_START + \
```

Comment:
minor: `TZDRAM_BASE + TZDRAM_SIZE` is shorter.

Reply:
ok.
would it make sense to move this part of the defined back to platform_config.h and leave only soc specific config in the soc config files?

Reply:
Constraints on secure/non-secure memory can be SoC specific. Let's see that later, together with https://github.com/OP-TEE/optee_os/pull/2275.